src/emulator/io/include/io/file_ressources.h

```diff
@@ -0,0 +1,70 @@
+// Vita3K emulator project
+// Copyright (C) 2018 Vita3K team
+//
+// This program is free software; you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation; either version 2 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License along
+// with this program; if not, write to the Free Software Foundation, Inc.,
+// 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+#pragma once
+#include <boost/filesystem.hpp>
+
+namespace fs = boost::filesystem;
+
+const std::string SIMPLE_SLASH = "/"; // Simple /, to prevent propagating "/" everywhere
+
+const std::string MOUNT_POINT = ":"; // Simple : for indicating mount point
+
+const std::string UX0 = "ux0"; // Value for the Vita's memory card location, or external user partition 
+
+const std::string MOUNTED_UX0 = UX0 + MOUNT_POINT; // Simple mounted ux0 -> ux0:
+
+// TODO: Find meaning of uma0, could not find
+const std::string UMA0 = "uma0"; // Value for the Vita's ??? location
+
+const std::string USER_00_FOLDER = "/00"; // Name of the folder that contains 00 for the user
+
+const std::string DATA_FOLDER = "/data"; // Name of the folder that contains data
+
+const std::string APP_FOLDER = "/app"; // Name of the folder that contains the applications
+
+const std::string USER_FOLDER = "/user"; // Name of the folder that contains the user data
+
+const std::string SAVEDATA_FOLDER = "/savedata"; // Name of the folder that contains the save data
+
+const std::string SHADERLOG_FOLDER = "/shaderlog"; // Name of the folder that contains the shader logs
```

Comment:
This string is hard-coded elsewhere in the codebase.
TODO: Do a search for it and replace with a function call from here, or use this constant.